### DISCUSSION TITLE
Copying a glyph from one project and pasting to another

### DISCUSSION THREAD

**Discussion by jacklythgoee** [fontforge/fontforge#5024] [Q&A] [answered]
Hi,
I have spent ages researching how to copy and paste a glyph from one font project to another. Is there a way i can copy the file of the Unicode and paste in the other project?
I have an arabic font that we have paid for that needs to be pasted into a font project i have created, they have the same unicode. But just need to get inside the other project file

**Comment by skef** [comment, marked as answer]
If you're asking about doing this in the UI, copy/paste works as long as you open both fonts in the same running program. That is, you can open one using the OS (e.g. double-clicking on a font file) but must open the other from the FontForge "File" menu of the first.

If you're asking about python or the native scripting language the answer is a bit more complicated.
